esmvaltool/diag_scripts/validation.py

```diff
@@ -239,10 +316,14 @@ def main(cfg):
             for obs_i, obsfile in zip(obs_list, obs):
                 obs_analyzed = coordinate_collapse(obs_i, cfg)
                 obs_name = obsfile['dataset']
-                plot_key = short_name + '_CONTROL_vs_' + obs_name
+                plot_key = short_name + '_' + control_dataset_name + \
```

Comment:
Minor code style remark: using f-strings would be a bit nicer than all the pluses and using brackets around multi  line expressions instead of backslashes would also be a bit nicer.

Reply:
le done (most of le everywhere)